src/main/java/org/jabref/logic/pdf/search/PdfIndexer.java

```diff
@@ -45,16 +48,34 @@ public class PdfIndexer {
     private static final Logger LOGGER = LoggerFactory.getLogger(PdfIndexer.class);
 
     @VisibleForTesting
+    @Nullable // null might happen if lock is held by another JabRef instance
```

Comment:
I am not a fan of this Nullable/non null annotations. Visible for testing is okay for me